client/my-sites/domains/components/form/country-select.jsx

```diff
@@ -76,6 +76,7 @@ export default React.createClass( {
 
 					<FormSelect
 						name={ this.props.name }
+						id={ this.props.name }
```

Comment:
A name is not necessarily a valid ID. What is the form of `this.props.name`?

Reply:
But since `this.props.name` is used by `FormLabel.htmlFor`, we can assume it is valid.